packages/translations/yarn.lock

```diff
@@ -9,58 +9,6 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@execonline-inc/collections@^2.1.0":
-  version "2.1.0"
-  resolved "https://registry.yarnpkg.com/@execonline-inc/collections/-/collections-2.1.0.tgz#b0b4b57a777a9822ec64e7e999601276c8a227c9"
-  integrity sha512-GhftZMh2Uzd3u3hqFxJblCPh1GZnS2fAvUVXIXsUdz26emYLIke/WO2sUOlMo/CYB9ork33+owQd8/cu2DVG5A==
-  dependencies:
-    "@execonline-inc/maybe-adapter" "^1.0.2"
-    "@kofno/piper" "^1.2.0"
-    ajaxian "^3.0.1"
-    jsonous "^5.0.1"
-    logging "^3.2.0"
-    maybeasy "^2.6.0"
-    resulty "^3.0.0"
-    taskarian "^2.1.0"
-
-"@execonline-inc/environment@^1.0.2":
-  version "1.0.2"
-  resolved "https://registry.yarnpkg.com/@execonline-inc/environment/-/environment-1.0.2.tgz#2ae93cf705023e6b770afead09c1a11bd6a8483f"
-  integrity sha512-07eoD9iwwyIJaO7Cx/CYNvUgxcRoFPnOHnulaVVuL+CcOTwvzUmB0K8jwARCXPjR7yigiogP20u0bGQCUpefMQ==
-  dependencies:
-    "@kofno/piper" "^1.2.0"
-    ajaxian "^3.0.1"
-    honeybadger-js "^2.0.0"
-    jsonous "^5.0.1"
-    logging "^3.2.0"
-    maybeasy "^2.6.0"
-    resulty "^3.0.0"
-    taskarian "^2.1.0"
-
-"@execonline-inc/logging@^1.1.0":
```

Comment:
However, `logging`, `collections`, and `maybe-adapter` are used by the translations package and shouldn't be removed...

Reply:
If you are using the lerna tool correctly, it should handle managing the cross package dependencies.

Reply:
Does this mean it's safe to remove these? @kofno

Reply:
I don't how you removed them. If you are in the root of the project and you run `lerna link`, if will create the correct links between dependencies. 

You can learn more about how to use lerna here --> https://github.com/lerna/lerna/tree/master/commands/link#readme

Reply:
Here is an update @kofno :

I cloned the repo on my new laptop and checked out to the branch. Here are the commands i ran in order

* `yarn install`
* `lerna link`
* `lerna bootstrap`
* `lerna run tsc`

After doing the above, the last command returns this error in the `appy` package
<img width="849" alt="Screenshot 2020-10-23 at 1 34 23 PM" src="https://user-images.githubusercontent.com/32465536/97005148-ec7d0b80-1535-11eb-82d6-9c29c06a8a89.png">


I had to `cd` into the package and run `yarn install` manually. Doing that didn't update the yarn lock file.

Running `lerna run tsc` again return a success this time around
<img width="695" alt="Screenshot 2020-10-23 at 1 34 52 PM" src="https://user-images.githubusercontent.com/32465536/97005287-251ce500-1536-11eb-90b9-2415d6f527d4.png">


How do we proceed now?

Reply:
I don't think you need to run lerna bootstrap... I think that creates links where they aren't needed.

Reply:
Also, I think each package will have its own yarn.lock too.

Reply:
When i run only `lerna link` without bootstrapping and then run `lerna run tsc`, i am getting this error 
<img width="828" alt="Screenshot 2020-10-23 at 2 29 44 PM" src="https://user-images.githubusercontent.com/32465536/97009657-44b70c00-153c-11eb-8595-01976cf2c500.png">

Reply:
Interesting... maybe I am wrong about bootstrap. But if I am, then I'm surprised you had do the piper install separately.

Well, go back to what you did before so it builds. Then we can just merge. These security issues are in libraries that are merely supportive. We don't release any code that relies on them, so these vulnerabilities haven't been released through our code.

Reply:
I don't recall the CircleCI build ever working but the will squash the commits into one

Reply:
@kofno This still needs your attention . Thank you

Reply:
No, circleCI has never worked. Does `lerna run tsc` work?

Reply:
Yes it does
<img width="863" alt="Screenshot 2020-11-02 at 3 01 36 PM" src="https://user-images.githubusercontent.com/32465536/97876589-58bcf380-1d1c-11eb-83e6-e2bd686a6d4c.png">